setup.py

```diff
@@ -121,7 +121,9 @@ def find_package_data(where=".", package="", exclude=standard_exclude,
     install_requires=['Django >= 2.0',
                       'python3-openid >= 3.0.8',
                       'requests-oauthlib >= 0.3.0',
-                      "requests"],
+                      "requests",
+                      "pyjwt >=1.7",
+                      "cryptography >= 2.8"],
```

Comment:
Is cryptography needed? I don't see it being referenced in your code?

Reply:
@pennersr sure it does, as it described [here](https://pyjwt.readthedocs.io/en/latest/installation.html#cryptographic-dependencies-optional), there is RS256 decoding requires `cryptography` package